source/contracts/reporting/Market.sol

```diff
@@ -30,7 +30,7 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
     using SafeMathInt256 for int256;
 
     // Constants
-    uint256 private constant MAX_FEE_PER_ETH_IN_ATTOETH = 1 ether / 2;
+    uint256 private constant MAX_FEE_PER_ETH_IN_ATTOETH = 15 * 10**16; // 15%
```

Comment:
Did you have an argument for why this is a reasonable number?  This feels way too low to put a hard-cap in for.  Credit cards frequently charge more than this (significantly in some cases), as an example of a situation where people are willing to pay greater than 15% in fees for something.

Reply:
Yea. When considering the maximum reporting fee this is near the highest we can allow the market creator fee before the combined fee makes participation irrational (combined maximum is now 48% where 50% mean someone would always be losing even if they win).